src/pages/settings/data.tsx

```diff
@@ -0,0 +1,55 @@
+import SettingsHeader from '@/components/settings/SettingsHeader'
+import { Paper, Grid, Typography, Tooltip, SvgIcon, Button } from '@mui/material'
+import InfoIcon from '@/public/images/notifications/info.svg'
+import type { NextPage } from 'next'
+import Head from 'next/head'
+import { useState } from 'react'
+import ImportAllDialog from '@/components/settings/ImportAllDialog'
+
+const Data: NextPage = () => {
+  const [modalOpen, setModalOpen] = useState(false)
+
+  return (
+    <>
+      <Head>
+        <title>Safe – Settings – Data</title>
+      </Head>
+
+      <SettingsHeader />
+
+      <main>
+        <Paper sx={{ p: 4, mb: 2 }}>
+          <Grid container spacing={3}>
+            <Grid item sm={4} xs={12}>
+              <Typography variant="h4" fontWeight={700}>
+                Data import
+                <Tooltip
+                  placement="top"
+                  title="The data import will overwrite all added Safes and all address book entries"
```

Comment:
Could this perhaps be an inline piece of text?
Similar to this design?

<img width="963" alt="Screenshot 2022-11-09 at 13 58 17" src="https://user-images.githubusercontent.com/381895/200836745-9db94618-888e-4a4a-bedc-ecbdb991b3aa.png">

cc @liliiaorlenko